parachains/runtimes/assets/statemint/src/xcm_config.rs

```diff
@@ -258,6 +263,8 @@ pub type Barrier = DenyThenTry<
 				AllowExplicitUnpaidExecutionFrom<ParentOrParentsPlurality>,
 				// Subscriptions for version tracking are OK.
 				AllowSubscriptionsFrom<ParentOrSiblings>,
+				// Fellows plurality gets free execution.
+				AllowExplicitUnpaidExecutionFrom<FellowsPlurality>,
```

Comment:
Huh, does this mean that even the root of Polkadot doesn't get free execution and only the technical fellowship does?

Reply:
Root should, but wouldn't `AllowExplicitUnpaidExecutionFrom<ParentOrParentsPlurality>` allow that?

Reply:
Ah, sorry I missed it -- it would seem like we should just have 1 `AllowExplicitUnpaidExecutionFrom` barrier instead of 2.